src/app/modules/forms/ix-forms/directives/with-glossary.directive.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  ComponentRef, Directive, TemplateRef, ViewContainerRef, AfterContentInit,
+} from '@angular/core';
+import { IxFormWithGlossaryComponent } from 'app/modules/forms/ix-forms/components/ix-form-with-glossary/ix-form-with-glossary.component';
+
+@Directive({
+  selector: '[withGlossary]',
```

Comment:
What's the advantage of making this a directive?
Could we not make it an ordinary component instead and let people place glossary in their forms themselve?